config/rubocop.yml

```diff
@@ -0,0 +1,56 @@
+require: rubocop-rspec
```

Comment:
@dmitrytrager I think your file has all the relevant rules included, i will delete the old file.